products/compute/terraform.yaml

```diff
@@ -1187,6 +1187,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateGCPName'
+  RegionNetworkEndpointGroup: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_network_endpoint_group_basic"
```

Comment:
This is no longer a basic test. I'd recommend having three separate tests instead of one mega-test.